<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExport</class>
 <widget class="QDialog" name="DialogExport">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose the columns that you would like to export:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_xLeftRough">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X left eye rough</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_yLeftRough">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y left eye rough</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_xRightRough">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X right eye rough</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_yRightRough">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y right eye rough</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_xFilter">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtered X</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_yFilter">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtered Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rough data:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_LeftPupil">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pupil dilation Left</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_rightPupil">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pupil dilation Right</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtered data:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fixations:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_fixNumber">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fixation number</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_fixDur">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration (seconds)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_xAverage">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Average X</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_yAverage">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Average Y</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_euclideanDis">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>RMS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_cancel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_export">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Export Current</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Saccades:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>For timestamped data, fixation and saccade data is presented alongside the data rows which encompass that event.  Please note that the saccades calculation may only be accurate in high quality datasets</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_sacNumber">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Saccade number</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_sacDuration">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration (seconds)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_smoothpursuit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Is smooth pursuit</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_sacAmplitude">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>280</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Amplitude</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_sacVelocityAv">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Average Velocity</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_sacVelocityPeak">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>340</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Peak Velocity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_export_all">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>430</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Export All</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_velocity">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Velocity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_select_all">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_timestamped">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Timestamped Data</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_fixations">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fixations</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_saccades">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Saccades</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
